whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -156,6 +161,80 @@ public Object mapSearch(final String requestUrl, final List<RpslObject> objects,
         return mapCommons(searchResult, requestUrl);
     }
 
+    public Object mapOrganisationEntity(final String requestUrl, Iterator<RpslObject> objects,
+                                        final RpslObjectDao objectDao, final int maxResultSize){
+
+        final List<Autnum> autnums = Lists.newArrayList();
+        final List<Ip> ipv4Networks = Lists.newArrayList();
+        final List<Ip> ipv6Networks = Lists.newArrayList();
+        RdapObject organisation = new RdapObject();
+        for (Iterator<RpslObject> it = getTopLevelObjects(objects); it.hasNext(); ) {
+            final RpslObject object = it.next();
+
+            switch (object.getType()) {
+                case AUT_NUM:
+                    autnums.add((Autnum) getRdapObject(requestUrl, object, objectDao.getLastUpdated(object.getObjectId()),
+                            Optional.empty()));
+                    break;
+                case INETNUM:
+                    ipv4Networks.add((Ip) getRdapObject(requestUrl, object, objectDao.getLastUpdated(object.getObjectId()),
+                            Optional.empty()));
+                    break;
```

Comment:
Inetnum and inet6num is doing the same thing, We should not use two different lists for same IP class